File-Formats/Presentation/create-read-edit-powerpoint-files-in-blazor.md

```diff
@@ -362,8 +362,8 @@ Step 9: Add the following JavaScript function in the _Host.cshtml in the Pages f
 
 By executing the program, you will get the **PowerPoint document** as follows.
 
-![Blazor Client-Side output PowerPoint document](Workingwith_Blazor/Output.png)
+![Blazor WASM output PowerPoint document](Workingwith_Blazor/Output.png)
 
-N> Even though PowerPoint library works in client-side, it is recommended to use server-side deployment. Since the client-side deployment increases the application payload size. You can also explore our [Blazor PowerPoint library demo](https://blazor.syncfusion.com/demos/powerpoint/getting-started) that shows how to create and modify PowerPoint files from C# with just five lines of code.
+N> Even though PowerPoint library works in WASM, it is recommended to use server deployment. Since the WASM deployment increases the application payload size. You can also explore our [Blazor PowerPoint library demo](https://blazor.syncfusion.com/demos/powerpoint/getting-started) that shows how to create and modify PowerPoint files from C# with just five lines of code.
```

Comment:
Even though PowerPoint library works in WASM app, it is recommended to use server deployment. Since the WASM app deployment increases the application payload size.